doc/lsst.synpipe/index.rst

```diff
@@ -0,0 +1,24 @@
+.. py:currentmodule:: lsst.synpipe 
+
+.. _lsst.synpipe:
+
+############
+lsst.synpipe
+############
+
+.. Paragraph that describes what this Python module does and links to related modules and frameworks.
+
+.. Add subsections with toctree to individual topic pages.
+
+Using lsst.synpipe
+==================
+
+.. toctree::
+   :maxdepth: 1
+
+   Synpipe_QuickStart.rst
```

Comment:
Please rename the file `Synpipe_QuickStart.rst` to `quick-start.rst`. I know I haven't published URL recommendations yet, but this will be it :)